src/docs/database-migrations.mdx

```diff
@@ -376,3 +376,60 @@ Renaming columns is dangerous and will result in downtime. The reason this occur
   - Stop writing to the old column and remove references from the code.
   - Drop the old column from the database.
   - Generally, this is not worth doing and a lot of risk/effort compared to the reward.
+
+## Migration Deployment
+
+<Alert title="Employees Only" level="warning">
+  The following section covers migration deployment in Sentry's
+  saas products.
+</Alert>
+
+We support two kinds of migrations in our saas deployments:
+
+- Deploy time migrations
+- Post-deploy migrations
+
+### Deployment Migrations
+
+Deployment migrations are run in each region and tenant before code is deployed. Deployment migrations are expected to finish quickly and all statements must complete within 5 seconds. If a migration could take longer because a large number of rows is being operated on, it should be deployed as a post-deploy migration instead. Deployment migrations are ideal for:
```

Comment:
I don't totally remember, but I thought our timeout here was 2 seconds

Reply:
We have 5s in the [current production](https://github.com/getsentry/getsentry/blob/master/getsentry/conf/settings/regionsilo.py#L1409-L1410) config.